.github/workflows/dormant_issues_prs.yml

```
@@ -21,22 +21,35 @@ jobs:
           stale-pr-label: ":sleeping: Dormant"
           remove-stale-when-updated: true
           stale-issue-message: >
-            Hey, there hasn't been any activity on this issue for more than 180
-            days. For now, we have marked it as "dormant" until there is some
-            new activity. You are welcome to reach out to people by
-            mentioning them here or on our
+            Hello scikit-image core devs! There hasn't been any activity on
+            this issue for more than 180 days. I have marked it as "dormant" to
+            make it easy to find. You can remove the dormant tag by adding some
+            new activity.
```

```suggestion
            make it easy to find. The label will be removed automatically if
            if new comments are made.
```
It's not activity in general but comments only I think. Any way, if maintainers want to remove the label, it's faster to do it the conventional way. I wouldn't mind removing this last sentence or moving it to the next paragraph because this fact is more useful to people without label permissions.

Same below.